### PR TITLE
updated log message in create_instance in case of a NetworkError

### PR DESCRIPTION
**Pull request by gerva**
The output log is sent with the periodic email. To make emails more actionable, we should report that a NetworkError has just happened (it always happens when we try to connect, for the first time to the new instance). Just log the error message, instead of the full "exc_info"

### CODE REVIEW

File: cloudtools/scripts/aws_create_instance.py [hunk 1]
```diff
@@ -163,7 +165,17 @@ def create_instance(name, config, region, key_name, ssh_key, instance_data,
                                 ssh_key=ssh_key, instance_data=instance_data,
                                 deploypass=deploypass, reboot=reboot)
             break
+        except NetworkError as e:
+            # it takes a while for the machine to start/reboot so the
+            # NetworkError exception is quite common, just log the error,
+            # without the full stack trace
+            log.warn("problem assimilating %s (%s, %s) - %s, retrying in "
```
Comment (djmitche) [inline cloudtools/scripts/aws_create_instance.py]: This message should be adjusted -- something like "cannot connect; instance may still be starting"

Comment (selenamarie) [inline cloudtools/scripts/aws_create_instance.py]: +1 "problem assimilating" isn't clear

**Comment by gerva** [comment]
bah I forgot a double quote!
I did not run tox  because I don't have MySQL installed on my machine and `./tox` fails with:
```
Collecting MySQL-python==1.2.5 (from build-cloud-tools==1.0.0)
  Using cached MySQL-python-1.2.5.zip
    Complete output from command python setup.py egg_info:
    sh: mysql_config: command not found
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/private/var/folders/n6/9ldd1kwd48d8wdlxhth4hvt80000gn/T/pip-build-7qjbsm/MySQL-python/setup.py", line 17, in <module>
        metadata, options = get_config()
      File "setup_posix.py", line 43, in get_config
        libs = mysql_config("libs_r")
      File "setup_posix.py", line 25, in mysql_config
        raise EnvironmentError("%s not found" % (mysql_config.path,))
    EnvironmentError: mysql_config not found
```

Could we make the MySQL-python dependency optional for tests?

**Comment by djmitche** [comment]
Sounds fine to me, as long as it still installs correctly in production - so probably with `extras_require`.

**Comment by djmitche** [comment]
patch looks good now!

**Pull request merged**